CycloneDX.Core.Tests/ComponentServiceTests.cs

```diff
@@ -14,15 +14,14 @@
 //
 // Copyright (c) Steve Springett. All Rights Reserved.
 
-using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
-using Xunit;
-using CycloneDX.Models;
 using CycloneDX.Core.Models;
+using CycloneDX.Models;
 using CycloneDX.Services;
 using Moq;
+using Xunit;
 

```

Comment:
Don't submit changes like this. Just noise without any value.

Reply:
While submitting PR, I forgot to run tests and I don't know how the the tests projects usings was removed, so I restored them back, then ran tests and then committed the changes!
This is just rearranging usings, not much! 😇